ivy/functional/frontends/tensorflow/activations.py

```diff
@@ -22,3 +22,46 @@ def sigmoid(x):
 
 def softmax(x, axis=-1):
     return ivy.softmax(x, axis=axis)
+
+
+def elu(x, alpha=1.0):
+    zeros = ivy.zeros_like(x)
+    ones = ivy.ones_like(x)
+    ret_val = ivy.where(
+        x > zeros, x, ivy.multiply(alpha, ivy.subtract(ivy.exp(x), ones))
+    )
+    return ret_val
+
+
+elu.unsupported_dtypes = (

```

Comment:
Adding unsupported_dtype to functions should ideally be done in the form of a dictionary Kindly refer to the Frontends section in the Deep Dive guide for more details.